pylib/cqlshlib/cql3handling.py

```diff
@@ -380,6 +380,11 @@ def dequote_value(cqlword):
                             ( ender="," [propmapkey]=<term> ":" [propmapval]=<term> )*
                       ender="}"
                     ;
+<propertyOrOption> ::= <property>
```

Comment:
Maybe having a standalone `copyOption` for `ALL | INDEXES | TRIGGERS` and turn this into
```
<propertyOrOption> ::= <property>
                                       | <copyOption>
                                      ;
```
is cleaner. What do you think?